### ISSUE TITLE
flat-list render-prop-fn, property or method "args" is not defined

### ISSUE THREAD

**Issue by SunPuzzle** [GeekyAnts/vue-native-core#234]
### flat-list, render-prop-fn not work with conditional rendering

Get errors in console:
```
[Vue warn]: Property or method "args" is not defined on the instance but referenced during render. Make sure to declare reactive data properties in the data option.
```
also
```
TypeError: renderItem is not a function
- node_modules/react-native/Libraries/Lists/FlatList.js:626:13 in FlatList._this._renderItem
- node_modules/react-native/Libraries/Lists/VirtualizedList.js:1733:20 in CellRenderer.render
```

```
<view v-if="1==0">
    <text>Hello world</text>
</view>
<view v-else>
    <flat-list
            :style="{backgroundColor: 'red', height: 200, width: 200, overflow: 'hidden'}"
            :data="[{key: 'a'}, {key: 'b'}]" >
        <view render-prop-fn="renderItem" >
            <text>{{args.item.key}}</text>
        </view>
    </flat-list>
</view>
```

- Expo: yes
- OS: macOS 10.15.2 
- Device: Emulator Nexus_5X_API_26_x86
- run with "npm run android"
```
"expo": "^35.0.0",
"react-dom": "16.8.3",
"react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
```

**Comment by SunPuzzle** [comment]
possible solution, make component SomeList.vue
```
<template>
    <flat-list
            :style="{backgroundColor: 'red', height: 200, width: 200, overflow: 'hidden'}"
            :data="[{key: 'a'}, {key: 'b'}]" >
        <view render-prop-fn="renderItem" >
            <text>{{args.item.key}}</text>
        </view>
    </flat-list>
</template>
```
```
<template>
<view v-if="1==0">
    <text>Hello world</text>
</view>
<view v-else>
    <SomeList/>
</view>
</template>

<script>
    import SomeList from "../components/SomeList";

export default {
 components: {
   SomeList
 }
}
</script>
```

**Comment by RishabhKarnad** [comment]
Hey @SunPuzzle, this might be an issue with the usage, but we'll look into it

**Comment by RishabhKarnad** [comment]
Fixed in [v0.3.0-rc.2](https://github.com/GeekyAnts/vue-native-core/releases/tag/v0.3.0-rc.2)
